src/components/menu-surface/menu-surface.tsx

```diff
@@ -61,7 +61,7 @@ export class MenuSurface {
     }
 
     public componentDidUpdate() {
-        this.ensureMenuFitsInViewPort();
+        this.createResizeObserver(this.host);
```

Comment:
Do we have to create a new observer every time the component is updated? :thinking: We should be able to just create one and then remove it when the component is destroyed

Reply:
I also thought about that, now the observer is removed in the timeout function. I don't know what is best, only have it when it is resizing(updating) or have it there all the time? but do you think it is better that the observer is always there? 🙂

Reply:
It will allow us to remove the ugly `setTimeout` :sweat_smile:

Reply:
haha 😆 I'll do a fix-up then!